guides/doc-Upgrading_and_Updating/topics/upgrading_overview.adoc

```diff
@@ -63,35 +49,56 @@ For more information, see the Red Hat Knowledgebase solution https://access.redh
 
 ifdef::satellite[]
 You can upgrade to {ProjectName} {ProjectVersion} from {ProjectName} {ProjectVersionPrevious}.
+
 {ProjectServer}s and {SmartProxyServer}s on earlier versions must first be upgraded to {Project} {ProjectVersionPrevious}.
-For more information, see the {Project} {ProjectVersionPrevious} https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProjectVersionPrevious}/html/upgrading_and_updating_red_hat_satellite/[Upgrading and Updating {ProjectName}] guide.
+For more information, see the https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProjectVersionPrevious}/html/upgrading_and_updating_red_hat_satellite/[_{UpgradingDocTitle} to {ProjectVersionPrevious}_].
 endif::[]
 
 ifndef::satellite[]
 You can upgrade to {ProjectName} {ProjectVersion} from {ProjectName} {ProjectVersionPrevious}.
+
 {ProjectServer}s and {SmartProxyServer}s on earlier versions must first be upgraded to {Project} {ProjectVersionPrevious}.
 For more information, see the {Project} {ProjectVersionPrevious} Upgrade documentation.
 endif::[]
 
-.Overview of {Project} {ProjectVersion} Upgrade Paths
-ifdef::satellite[]
-image::satellite_6.4_upgrade_paths.png[Overview of {Project} {ProjectVersion} Upgrade Paths]
-
-WARNING: Upgrading from the Beta to GA version is not supported.
-endif::[]
+=== High-Level Upgrade Steps
 
-The high level steps in upgrading to {Project} {ProjectVersion} are as follows.
+The high-level steps in upgrading {Project} to {ProjectVersion} are as follows.
 
 ifdef::satellite[]
 . Clone your existing {ProjectServer}s. For more information, see xref:cloning_satellite_server[].
 endif::[]
 . Upgrade {ProjectServer} to {ProjectVersion}.
 For more information, see xref:Upgrading_Server_{context}[].
-. Upgrade all {SmartProxyServer}s to {ProjectVersion}. For more information, see xref:upgrading_capsule_server[].
+. Upgrade all {SmartProxyServer}s to {ProjectVersion}.
+For more information, see xref:upgrading_capsule_server[].
 ifdef::katello,orcharhino,satellite[]
-. Upgrade to {project-client-name} on all {Project} clients.
-For more information, see xref:upgrading_clients[].
+. Upgrade to {project-client-name} on all content hosts.
+For more information, see xref:upgrading_content_hosts[].
+endif::[]
+ifdef::satellite[]
+. After you upgrade your {Project}, you have to upgrade the operating system on your {ProjectServer}s and {SmartProxies} to {RHEL} 8.
+endif::[]
+ifndef::satellite,foreman-deb,orcharhino[]
+. After you upgrade your {Project}, you have to upgrade the operating system on your {ProjectServer}s and {SmartProxies} to Enterprise Linux 8.
```

Comment:
Same-ish comment as in the Satellite case, but with the added complexity that upstream users might already be on EL8 (I had forgotten about this detail when talking to you earlier).

So, maybe like this?
After you upgrade your {Project} *and it's running on Enterprise Linux 7*, you *can* upgrade the operating system on your {ProjectServer}s and {SmartProxies} to Enterprise Linux 8.